utils/env_preferences/src/parse/mod.rs

```diff
@@ -0,0 +1,16 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+//! Parsing functionality for various popular operating systems
+
+// Re-export all alias functions
+mod aliases;
+pub use aliases::*;
```

Comment:
why are we making these public?

Reply:
Based on [this comment](https://github.com/unicode-org/icu4x/pull/6158#discussion_r1966270208) by @zbraniecki, but happy to change it if that's preferred!